stable/datadog/templates/agent-service-metrics.yaml

```diff
@@ -2,10 +2,10 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ template "datadog.clusterAgent.fullname" . }}-metrics-api
+  name: {{ include "datadog.fullname" . }}-cluster-agent-metrics-api
   labels:
     app: "{{ template "datadog.fullname" . }}"
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    chart: {{ include "datadog.chart" . }}
```

Comment:
The selector with `{{ template "datadog.clusterAgent.fullname" . }}` needs to be updated too https://github.com/helm/charts/pull/11903/commits/e8e3405a850ea2294f73d8557c4329409fed4280#diff-c830c155f9433d8d2cb2520cf4b3e680R14